cibuild.cmd

```diff
@@ -107,3 +100,14 @@ exit /b 0
 @echo            with /release. May not be combined with /test32 or /test64.
 @echo.
 @goto :eof
+
+:BuildFailed
+echo Build failed with ERRORLEVEL %ERRORLEVEL%
+call :TerminateCompilerServer
+exit /b 1
+
+:TerminateCompilerServer
+@REM Kill any instances VBCSCompiler.exe to release locked files, ignoring stderr if process is not open
+@REM This prevents future CI runs from failing hile trying to delete those files.
+
+taskkill /F /IM vbcscompiler.exe 2> nul
```

Comment:
@davkean, We should only kill instances of VBCSCompiler spawned from the Compiler Toolset. Instances of VBCSCompiler from MSBuild or from the just built bits, could be potentially in use by a test or a separate job on the machine (we aren't the only ones to run on the open pool).